plugin-server/src/main/ingestion-queues/session-recording/session-recordings-consumer.ts

```diff
@@ -313,6 +319,8 @@ export class SessionRecordingIngester {
             )
         }
 
+        sessionInfoSummary.observe(event.metadata.rawSize)
```

Comment:
we already nearby check the library version and raise an ingestion warning if the session is from a version before we added sensible batching

i was trying to avoid dropping data since older SDK doesn't automatically mean a bad session... but we could push every session from before some arbitrary version earlier than those to overflow instead of dropping them

or apply lower thresholds to them so they're more likely to overflow

this is anyway an interesting metric to gather, but we _also_ have the version signal to let us make a better decision

Reply:
That's indeed a valid option, I wanted to explore a future-proofing option of characterizing the behaviour instead: it could come back with an edge case of a future release, or if someone tries to be smart and tweak the SDK / submit events manually.